packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -3596,6 +3602,42 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
     }
   );
 
+  getLinkHoverTooltip = memoize(
+    (linkHoverTooltipProps: CSSProperties): ReactNode => {
+      if (linkHoverTooltipProps == null) {
+        return null;
+      }
+
+      const { linkHoverDisplayValue } = this.state;
+
+      const wrapperStyle: CSSProperties = {
+        position: 'absolute',
+        ...linkHoverTooltipProps,
+        pointerEvents: 'none',
+      };
+
+      const popperOptions: PopperOptions = {
+        placement: 'bottom',
+      };
+
+      return (
+        <div style={wrapperStyle}>
+          <Tooltip
+            key={Date.now()}
+            options={popperOptions}
+            ref={this.handleTooltipRef}
+          >
+            <div style={{ textAlign: 'left' }}>
+              {linkHoverDisplayValue} - Click once to follow.
+              <br />
+              Click and hold to select this cell.
+            </div>
```

Comment:
It might be preferable to do what Google sheets does: the tooltip has links and you click the link in the tooltip. Regular click in the cell just selects the cell

I think that might simplify some things. Also, the link handling might only need to be in `IrisGrid` since `Grid` is supposed to be fairly minimal/no frills

Reply:
We intentionally follow excel style behaviour for link display and interaction. We discussed both when making the ticket and decided we like the immediacy of being able to directly click a link.

Reply:
Sounds good. What about removing the link from the tooltip then? The tooltip just feels very large. And the link text distracts from the interaction explanation which I feel should be the primary focus of a non-interactive tooltip